src/qt/res/bitcoin-qt-res.rc

```diff
@@ -19,7 +19,7 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoin Core (OSS GUI client for Bitcoin)"
+            VALUE "FileDescription",    "Bitcoin Core (GUI client for Bitcoin)"
```

Comment:
Ugh, we have to client->node here to, to be consistent with the bitcoind change ("GUI Bitcoin node"?)